common/src/main/java/com/sequenceiq/cloudbreak/quartz/JobDataMapProvider.java

```diff
@@ -0,0 +1,35 @@
+package com.sequenceiq.cloudbreak.quartz;
+
+import org.quartz.JobDataMap;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.quartz.model.JobResourceAdapter;
+
+@Component
+public class JobDataMapProvider {
+
+    public static final String APP_VERSION_KEY = "appVersion";
+
+    private final String appVersion;
+
+    public JobDataMapProvider(@Value("${info.app.version:}") String appVersion) {
+        this.appVersion = appVersion;
+    }
+
+    public JobDataMap provide(JobResourceAdapter<?> jobResourceAdapter) {
```

Comment:
where is this used?

Reply:
com/sequenceiq/cloudbreak/job/stackpatcher/ExistingStackPatcherJobService.java:88

Reply:
my bad, did not recognize